include/multipass/virtual_machine.h

```diff
@@ -33,15 +33,15 @@ class VirtualMachine
 public:
     enum class State
     {
+        unknown,
```

Comment:
These are saved as numbers in the json instance db, so changing the order here will alter the meaning there. So, for instance, an instance that was saved as `suspended` would now be loaded as `suspending`, no?

Reply:
Oh, hmm, I think you're right.  I'd really like for `unknown` to be the first & default state in the enum, but if this breaks things, then I'm outta luck.  I'll try it to make sure though.

Reply:
Yep, you are definitely right, so we'll just live with this `enum` the way it is and can only add states at the end in order not to break anything.  That said...fixed.

Reply:
Yeah another instance where epochs would come in handy... An alternative would be to detect the version (e.g. w/ some slight change in the json that we could recognize) and interpret the state differently, but that would mean adding historical complexity and I am not sure it would be worth it...

Reply:
Yeah, definitely not worth it.